### DISCUSSION TITLE
Migrate from cloud to self-hosted

### DISCUSSION THREAD

**Discussion by dup2** [tolgee/tolgee-platform#2480] [Q&A]
Hi everyone,

We are evaluating tolgee for a project and would like to start now with the cloud hosted option but maybe later switch to a self-hosted installation.

Is this migration possible with full history and users? We see that the export and cli is mainly for integrating the final localisations into the product but not including all the metadata.

Thanks for your feedback!

**Comment by JanCizmar** [comment]
Hello! Thanks for the query. Unfortunately, complete export is not possible yet. You can only export data to .json and .xliff, but screenshots and metadata have to migrated manually or using some custom script you would have to implement.
